covid/transform.py

```diff
@@ -87,10 +87,10 @@
 # Criteria Category 3 Fields.
 MAX_ICU_BED_OCCUPATION_7_DAYS = "max_icu_bed_occupation_7_days"
 MAX_INPATIENT_BED_OCCUPATION_7_DAYS = "max_inpatient_bed_occupation_7_days"
-ICU_PERCENT_OCCUPIED = "icu_percent_occupied"
-INPATIENT_PERCENT_OCCUPIED = "inpatient_bed_percent_occupied"
-BASE_ICU_BEDS_FIELD = "% of ICU Beds Occupied"
-BASE_INPATIENT_BEDS_FIELD = "% of Inpatient Beds Occupied"
+ICU_PERCENT_OCCUPIED = "pct_icu_bed_utilization"
```

Comment:
@rypan @wesmwoo what should these column titles be now? Is there a new place to upload the hospital data as well?

Reply:
Reverted column names